Gemfile

```diff
@@ -3,7 +3,7 @@
 source "https://rubygems.org"
 
 gemspec
-rails_version = "#{ENV['RAILS_VERSION'] || '6.0.3.3'}"
+rails_version = (ENV["RAILS_VERSION"] || "6.0.3.3").to_s
```

Comment:
Mind updating this to `6.1.0`? We can also do so in another PR ❤️

Reply:
I can do it in a separate PR <3